src/modelfree/shaping_wrappers.py

```diff
@@ -0,0 +1,208 @@
+from collections import defaultdict
+import json
+
+import numpy as np
+from stable_baselines import logger
+from stable_baselines.common.base_class import BaseRLModel
+from stable_baselines.common.vec_env import VecEnvWrapper
+
+from modelfree import envs  # noqa - needed to register sumo_auto_contact
+from modelfree.scheduling import LinearAnnealer
+
+
+class RewardShapingEnv(VecEnvWrapper):
```

Comment:
Consider renaming to `RewardShapingVecWrapper`: it's not an `Env`, it's a `VecEnvWrapper`.

Reply:
sure